index.html

```diff
@@ -239,8 +144,11 @@ <h2>Contact me</h2>
                 
             </div>
         </div>
-        
     </footer>
+
+    <!-- Modal -->
+    <div id="modal-section"></div>
```

Comment:
[ ] The project's title and the list of technologies are missing. kindly add them to match the Figma design.
Your design : 
![popupte3ou](https://user-images.githubusercontent.com/62156607/208742724-a31dadc5-988a-4b0d-b74d-c37c66f314bf.PNG)

Figma design : 
![popupdesign](https://user-images.githubusercontent.com/62156607/208742769-a0b278e9-34d6-43e7-bb3c-ee20ebe97017.PNG)

Reply:
Thank You @nedjwab. So I can actually see the missing content you mention here on my end, I really don't know why you can not see them.

could it be because I updated the changes after requesting for review?

![image](https://user-images.githubusercontent.com/43586022/208743999-fab8028f-a2da-4a8b-814d-c5aeba0de8a2.png)